jib-core/src/test/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilderTest.java

```diff
@@ -84,12 +84,12 @@ private static void verifyNextTarArchiveEntryIsDirectory(
     assertThat(extractionPathEntry.getMode()).isEqualTo(TarArchiveEntry.DEFAULT_DIR_MODE);
   }
 
-  private static FileEntry defaultLayerEntry(Path source, AbsoluteUnixPath destination) {
+  private static FileEntry defaultLayerEntry(Path source, AbsoluteUnixPath destination) throws IOException {
     return new FileEntry(
         source,
         destination,
         FileEntriesLayer.DEFAULT_FILE_PERMISSIONS_PROVIDER.get(source, destination),
-        FileEntriesLayer.DEFAULT_MODIFICATION_TIME);
+        Files.getLastModifiedTime(source).toInstant());
```

Comment:
Is it possible to leave this in as `FileEntriesLayer.DEFAULT_MODIFICATION_TIME`? I think this helper method was initially meant to create a FileEntry with fixed permissions and modification time for testing purposes.

Reply:
The reason that this slipped through the test suite was partly because of this default value, actually.  The logic that should reset the modification time header wasn't tested because the tests reset it automatically here.  I've added a clarifying comment and also some more helpers to make it easier to add test layers with the actual mod-time of the temp files.